plugins/armadactl.yml

```
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: armadactl
+spec:
+  version: v0.3.8655
```

```suggestion
  version: v0.3.88
```

Can you look into ways to get this plugin manifest updated with our latest release?  Maybe as a followup.

---

After you apply this suggestion I think this is mergable actually.